prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -749,10 +750,10 @@ def _retrieve_context(output, doc_id, vector_index, answer) -> str:
     nodes = retriever.retrieve(answer)
     text = ""
     for node in nodes:
-        if node.score > 0.6:
+        if node.score > 0:
             text += node.get_content() + "\n"
         else:
-            app.logger.info("Node score is less than 0.6. " f"Ignored: {node.score}")
+            app.logger.info("Node score is less than 0. " f"Ignored: {node.score}")
```

Comment:
Is this possible @gaya3-zipstack ? I assumed its a value between 0 to 1

Reply:
@chandrasekharan-zipstack I am not very aware of this. I just added this quick fix here. Need to dig deeper.  For now,  keeping the changes minimal. Arun has suggested to revisit this later to add a more relevant check. SO for now, keeping the check in place.

Reply:
I suggest adding a comment around it or a todo to review it. We'll end up questioning this piece of code later

Reply:
Done.